src/workflows-app/components/OutputsTable.js

```diff
@@ -40,86 +39,80 @@ const OutputsTable = (props) => {
     );
   };
 
-  return h(AutoSizer, [
-    ({ width, height }) => {
-      return h(FlexTable, {
-        'aria-label': 'output-table',
-        rowCount: outputTableData.length,
-        sort: outputTableSort,
-        readOnly: false,
-        height,
-        width,
-        columns: [
-          {
-            size: { basis: 250, grow: 0 },
-            field: 'taskName',
-            headerRenderer: () =>
-              h(Sortable, { sort: outputTableSort, field: 'taskName', onSort: setOutputTableSort }, [h(HeaderCell, ['Task name'])]),
-            cellRenderer: ({ rowIndex }) => {
-              return h(TextCell, { style: { fontWeight: 500 } }, [outputTableData[rowIndex].taskName]);
-            },
+  return div({ style: { flex: '1 1 auto' } }, [
```

Comment:
It seems that Inputs table has `1 0 auto` instead. Should this too?

Reply:
Good catch, yes I think you're right. Thanks!